tests/helper.py

```diff
@@ -26,11 +28,26 @@ def clean_outs():
     if os.path.exists(sqliteout):
         os.remove(sqliteout)
 
-def tables_exist(db, tables):
-    """Checks if hdf5 database contains the specified tables.
+def which_outfile():
+    """Uses sqlite if platform is Mac, otherwise uses hdf5
     """
-    return all([t in db.root for t in tables])
+    return h5out if platform.system() == 'Linux' else sqliteout
 
+def tables_exist(outfile, tables):
+    """Checks if output database contains the specified tables.
+    """
+    if outfile == h5out:
+        db = tables.open_file(outfile, mode = "r")
```

Comment:
this should use a context manager like `with tables.open_file() as f:` or close the file explicitly